website/docs/advanced/targeting.md

```diff
@@ -194,6 +194,67 @@ The exact value that will be served to the users that fall into that fraction. D
 | Whole Number   | Integer      | any whole number within the range of `Int32`      |
 | Decimal Number | Double       | any decimal number within the range of `double` |
 
+## Stickiness & Consistency
+
+The percentage-based targeting is sticky by design and consistent across all SDKs.
+
+Percentage-based targeting is based on the identifier passed in the `User Object` to get `getValue()` methods in the SDKs. 
```

Comment:
Percentage-based targeting is based on the identifier of the `User Object` passed to SDK's `getValue()` methods.

Reply:
Fixed